diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -3155,24 +3180,42 @@ export class ContentGame extends React.Component {
                     </div>
                 )}
                 {suggestionType !== null && (suggestionType & UTILS.SuggestionType.MOVE) === UTILS.SuggestionType.MOVE && (
-                    <ChatContainer
-                        style={{
-                            display: "flex",
-                            border: "1px solid black",
-                            boxSizing: "border-box",
-                        }}
-                    >
-                        <ConversationHeader>
-                            <ConversationHeader.Content
-                                userName={`Moves Advice for ${engine.phase}`}
-                            />
-                        </ConversationHeader>
-
-                        <MessageList>
-                            {fullSuggestionComponent}
-                            {partialSuggestionComponent}
-                        </MessageList>
-                    </ChatContainer>
+                    <div>
+                    <div>
+                        <Button
+                            title={"Get ally-based advice"}
+                            color={"primary"}
+                            onClick={() => {
+                                if (latestMoveSuggestionFull) {
+                                    this.handleRecipientAnnotation(latestMoveSuggestionFull.time_sent, "replace");
+                                }
+                                
+                                this.sendMessage(
+                                    engine.client,
+                                    "GLOBAL",
+                                    `${JSON.stringify(this.state.stances)}`,
+                                    null,
+                                    null,
+                                    "move_advice_request",
+                                );
```

Comment:
I think that you accidentally added an extra parameter here:

```suggestion
                                this.sendMessage(
                                    engine.client,
                                    "GLOBAL",
                                    `${JSON.stringify(this.state.stances)}`,
                                    null,
                                    "move_advice_request",
                                );
```